packages/dev/core/src/DeviceInput/webDeviceInputSystem.ts

```diff
@@ -426,6 +429,10 @@ export class WebDeviceInputSystem implements IDeviceInputSystem {
                 pointer[PointerInput.Horizontal] = evt.clientX;
                 pointer[PointerInput.Vertical] = evt.clientY;
 
+                if (evt.pointerId === undefined) {
+                    evt.pointerId = this._mouseId;
+                }

```

Comment:
nit (suggestion)
```suggestion
                evt.pointerId =  evt.pointerId ?? this._mouseId;
```

Reply:
So when I tried this change, it didn't work for me because if this parameter exists as part of a browser created event, it will be readonly